runtime/doc/api.txt

```diff
@@ -1580,8 +1580,28 @@ nvim_set_hl({ns_id}, {name}, {*val})                           *nvim_set_hl()*
                              |nvim_create_namespace()|. Use 0 to set a
                              highlight group globally |:highlight|.
                     {name}   Highlight group name, e.g. "ErrorMsg"
-                    {val}    Highlight definition map, like |synIDattr()|. In
-                             addition, the following keys are recognized:
+                    {val}    Highlight definition map, accepts the following
+                             keys:
+                             • fg (or foreground): color name or "#RRGGBB",
```

Comment:
Small nit. All attributes are long form. Looks/reads better with long form first for background and foreground.

Reply:
I would argue that  "fg" and "bg" are more natural/important in this case due to the historical `guifg` & `ctermfg` in `:highlight` and I think 99% of colorschemes use `fg` and `bg` as key values.

Of course I will defer to the team, personally I was surprised to see the long forms were even accepted options.

Reply:
I'd say "fg" and "bg" are preferred, also consistent with ctermfg/ctermbg attrs which we also allow here.

Reply:
Actually I think I agree that it does *read* better, and "(or fg)" isn't any less clear. I will change it.

Reply:
Fixed, it also reduces some of the confusion around using `"fg"` as an actual value.

Reply:
Sorry @bfredl, I didn't see your comment. I will hold any re-revision until you guys confirm which style the team agrees on.